tools/populate_examiners_psc_rel.php

```diff
@@ -0,0 +1,213 @@
+        echo "examiners table contains NULL active statuses for examinerID: $row[examinerID]\n";
+    }
+}
+///////// examiners.full_name is null
+$pendingNullCheck = $db->pselect(
+    "SELECT * FROM examiners WHERE pending_approval IS NULL OR pending_approval =''",
+    array()
+);
+if (!empty($pendingNullCheck)) {
+    $errorPRE =true;
+    foreach ($pendingNullCheck as $k=>$row) {
+        echo "examiners table contains NULL pending approval statuses for examinerID: $row[examinerID]\n";
+    }
+}
+///////// certification.examinerID NOT in examiners.examinerID
+$certCheck = $db->pselect(
+    "SELECT DISTINCT examinerID FROM certification WHERE examinerID NOT IN (SELECT examinerID FROM examiners)",
+    array()
+);
+if (!empty($certCheck)) {
+    $errorPRE =true;
+    foreach ($certCheck as $k=>$row) {
+        echo "Certification table contains examiner ID: ".$row['examinerID']." which does not exist in the database (examiners).\n";
+    }
+}
+///////// instrument.Examiner NOT IN examiners.examinerID
+foreach ($instruments as $table=>$name) {
+    if ($db->tableExists($table) && $db->columnExists($table, 'Examiner')) {
+        $instCheck = $db->pselect(
+            "SELECT * FROM ".$table." WHERE Examiner NOT IN (SELECT examinerID FROM examiners)",
+            array()
+        );
+        if (!empty($instCheck)) {
+            $errorPRE =true;
+            foreach ($instCheck as $k=>$row) {
+                echo "Instrument $table contains examiner ID: ".$row['Examiner']." which does not exist in the database (examiners).\n";
+            }
+        }
+    }
+}
+///////// final_radiological_review.Final_Examiner2 final_radiological_review.Final_Examiner NOT in examiners.examinerID
+$frrCheck = $db->pselect(
+    "SELECT * FROM final_radiological_review 
+      WHERE Final_Examiner2 NOT IN (SELECT examinerID FROM examiners) 
+          OR Final_Examiner NOT IN (SELECT examinerID FROM examiners) ",
+    array()
+);
+if (!empty($frrCheck)) {
+    $errorPRE =true;
+    foreach ($frrCheck as $k=>$row) {
+        echo "Table final_radiological_review contains Final_Examiner IDs: ".$row['Final_Examiner']." and ".$row['Final_Examiner2']." which do not exist in the database (examiners).\n";
+    }
+}
+
+if ($errorPRE) {
+    echo "\n\nSome errors were detected, this script will now terminate.\nPlease Fix errors and try running script again.\n\n";
+    die();
+}
+
+foreach ($examinersRows as $k=>$row) {
+    $item = array(
+             'examinerID'       => $row['examinerID'],
+             'centerID'         => $row['centerID'],
+             'active'           => $row['active'],
+             'pending_approval' => $row['pending_approval'],
+            );
+    // Compare without spaces, without cases
+    $examinersOrganized[strtolower(str_replace(' ','',$row['full_name']))][] = $item;
+}
+
+print_r($examinersOrganized);
+
+echo "\nA copy of the `examiners` table has been created as `examiners_17.0`. \n\n";
```

Comment:
This is just reassuring the user that the data has been copied (in the SQL patch)